cmd/kogito/command/util/build_from_file.go

```diff
@@ -84,8 +84,11 @@ func ProduceTGZfile(resource string) (io.Reader, error) {
 		}
 		return nil
 	})
+	if err != nil{
+		log.Errorf("%s Error in walking the file", err)
```

Comment:
a message like "Error while walking through %s directory: %s, absoluteFilePath, err

Reply:
Sounds good.

Reply:
and move the the text to the message package, like this example: log.Infof(message.KogitoAppFoundFile, filesFound)

Reply:
alright then, let's keep it this way.